### PR TITLE
docs: update information about package management

### LINKED ISSUE #27853: REST API explain that Packages are created using a package manager
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28

### What part(s) of the article would you like to see updated?

There is no doc for creating new packages or editing permissions for an already existing one.

### Additional information

_No response_

----

_Maintainer update_

## Response to question on missing congtent

I checked in with an subject matter expert on this issue. They said:

> Packages can be published through client CLIs such as npm or docker, or through Actions, but not via the REST API.

## Content updates required to fix this issue

We can make this clearer to users by updating the introductory section for the REST API Packages reference docs. This part of the article is not automatically generated and can be fixed with a pull request to this repository. Any changes must be restricted to this section of the article: https://github.com/github/docs/blob/main/content/rest/packages/packages.md?plain=1#L17-L30.

Update the [About GitHub Packages](https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#about-github-packages) section as follows:

1. Revise the first paragraph to mention that GitHub Packages supports a range of package managers for publishing packages and that once you have published a package you can use the REST API to manage it.
2. Include a link to [Introduction to GitHub Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats) for more information.

### PR DESCRIPTION
**Pull request by Rizina**
### Why:

Closes: #27853

### What's being changed (if available, include any code snippets, screenshots, or gifs):

--

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/rest/packages/packages.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ autogenerated: rest
 
 ## About {% data variables.product.prodname_registry %}
 
-You can use the REST API to manage packages in your {% data variables.product.prodname_dotcom %} repositories and organizations. For more information, see "[AUTOTITLE](/packages/learn-github-packages/deleting-and-restoring-a-package)."
+Github packages suppports a range of package manager for publishing packages. For more information, see "[AUTOTITLE](/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats)."
```
Comment (steveward) [inline content/rest/packages/packages.md:19]:
```suggestion
{% data variables.product.prodname_registry %} suppports a range of package managers for publishing packages. For more information, see "[AUTOTITLE](/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats)."
```

**Review by steveward — DISMISSED** [review]
Thanks for the PR @Rizina! I'll get this triaged for review. ⚡

**Review by sabrowning1 — CHANGES_REQUESTED** [review]
👋🏼 @Rizina, thanks for opening another PR! ✨ I gave one small piece of feedback below; let me know if you have any questions 🙂

File: content/rest/packages/packages.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ autogenerated: rest
 
 ## About {% data variables.product.prodname_registry %}
 
-You can use the REST API to manage packages in your {% data variables.product.prodname_dotcom %} repositories and organizations. For more information, see "[AUTOTITLE](/packages/learn-github-packages/deleting-and-restoring-a-package)."
+{% data variables.product.prodname_registry %} suppports a range of package managers for publishing packages. For more information, see "[AUTOTITLE](/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats)."
```
Comment (sabrowning1) [inline content/rest/packages/packages.md:19]:
```suggestion
{% data variables.product.prodname_registry %} supports a range of package managers for publishing packages. For more information, see "[AUTOTITLE](/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats)."
```

Tiny typo fix!

**Review by sabrowning1 — APPROVED** [review]
Thanks for making that change @Rizina! I fixed one small typo, and this now looks good to go 🚀 I'll merge it shortly.

We really appreciate your continued help improving our docs! If you are looking for other ways to contribute and are up for a challenge, check out our [pumpkin-spice label](https://github.com/github/docs/issues?q=is%3Aissue+is%3Aopen+label%3A%22pumpkin-spice%22+) to find help wanted issues that are a bit more in depth.

**Pull request merged**